src/index.js

```diff
@@ -126,6 +161,8 @@ options.opt.locales = options.opt.locales.map(spec => {
     return loc.getSpec();
 });
 
+// make sure the mins and maxes are numeric
```

Comment:
leftover comment?

Reply:
Didn't get moved to line 40 along with the function. Now it is...